src/main/java/io/pinecone/unsigned_indices_model/QueryResponseWithUnsignedIndices.java

```diff
@@ -8,13 +8,43 @@
 import java.util.Collections;
 import java.util.List;
 
+/**
+ * This class represents the response to a query, where the `ScoredVector` objects in the `matches` list
+ * contain `SparseValuesWithUnsignedIndices` instead of the standard `SparseValues`.
+ * <p>
+ * The `QueryResponseWithUnsignedIndices` class contains the following fields:
+ * - `matches`: a list of `ScoredVectorWithUnsignedIndices` objects, representing the matching vectors
+ * - `namespace`: the namespace of the query
+ * - `usage`: the usage information for the query
```

Comment:
Defining a word by using the word (`usage`) is a little confusing. Looking at this, I'm not sure what `"usage"` means here. Does this mean, like, read and write units?

Reply:
Yes